packages/contract-dependencies-gnosis/src/ContractDependenciesGnosis.ts

```diff
@@ -127,6 +132,35 @@ export class ContractDependenciesGnosis extends ContractDependenciesEthers {
     this.gasPrice = gasPrice;
   }
 
+  async submitTransaction(transaction: Transaction<BigNumber>): Promise<TransactionReceipt> {
```

Comment:
This is so similar to the code in the parent class. Can we reduce the copy paste here somehow?